ros/api/v1/hosts.py

```diff
@@ -206,10 +186,12 @@ def build_system_filters():
         if operating_systems := request.args.getlist('os'):
             modified_operating_systems = []
             for os in operating_systems:
-                os = os.split(" ")[1]
-                if os not in OS_VERSIONS.keys():
-                    abort(400, message='Not a valid RHEL version')
-                modified_operating_systems.append(OS_VERSIONS[os])
+                os_object = {"name": "RHEL"}
```

Comment:
Observation:
With this filter change, it will break the current flow once merged without frontend pull-request. 
@r14chandra, could you please add a code block which will make this functionality backward compatible with FIXME comment? So that once both side changes (frontend + backend) related to os filtering pushed to Production, one can delete this code block.

Reply:
Sure adding a fix.